assessments/CLI_TX_KEA/earthmover.yml

```diff
@@ -0,0 +1,871 @@
+  # 1234       | math_v1      | Math                  | ...On Track            | txkea_blending                      | ...80
+
+{% set subjects_en = {
+    'txkea_dyslexia_checklist': {
+      'colname': 'txkea_dyslexia_checklist',
+      'descriptor': 'KG Dyslexia Referral Checklist',
+      'subject_assess_type':'checklist',
+      'subject_assess_lang': 'English',
+      'has_cp_col': True
+    },
+    'kea_dyslexia_screener': {
+      'colname': 'kea_dyslexia_screener',
+      'descriptor': 'Dyslexia Screener',
+      'subject_assess_type':'screener',
+      'subject_assessments': [
+        {'subject_assess_ref_1':[{'subject_asses_ref_colname':'txkea_letter_sounds', 'subject_asses_ref_descriptor':'Letter Sounds'}]},
+        {'subject_assess_ref_2':[{'subject_asses_ref_colname':'txkea_blending', 'subject_asses_ref_descriptor':'Blending'}]}
+      ],
+      'subject_assess_lang': 'English',
+      'has_cp_col': False
+    },
+    'kea_literacy_screening': {
+      'colname': 'kea_literacy_screening',
+      'descriptor': 'Literacy Screening',
+      'subject_assess_type':'screener',
+      'subject_assessments': [
+        {'subject_assess_ref_1':[{'subject_asses_ref_colname':'txkea_vocabulary_1', 'subject_asses_ref_descriptor':'Vocabulary Wave 1'}]},
+        {'subject_assess_ref_2':[{'subject_asses_ref_colname':'txkea_letter_names_1', 'subject_asses_ref_descriptor':'Letter Names Wave 1'}]},
+        {'subject_assess_ref_3':[{'subject_asses_ref_colname':'txkea_spelling_1', 'subject_asses_ref_descriptor':'Spelling Wave 1'}]}
+      ],
+      'subject_assess_lang': 'English',
+      'has_cp_col': False
+    },
+    'txkea_literacy_reading_composite': {
+      'colname': 'txkea_literacy_reading_composite',
+      'descriptor': 'Literacy Reading Composite',
+      'subject_assess_type':'composite',
+      'subject_assessments': [
+        {'subject_assess_ref_1':[{'subject_asses_ref_colname':'txkea_letter_names_1', 'subject_asses_ref_descriptor':'Letter Names Wave 1'}]},
+        {'subject_assess_ref_2':[{'subject_asses_ref_colname':'txkea_letter_sounds_1', 'subject_asses_ref_descriptor':'Letter Sounds 1'}]},
+        {'subject_assess_ref_3':[{'subject_asses_ref_colname':'txkea_blending_1', 'subject_asses_ref_descriptor':'Blending 1'}]}
+      ],
+      'subject_assess_lang': 'English',
+      'has_cp_col': False
+    },
+    'txkea_language_composite': {
+      'colname': 'txkea_language_composite',
+      'descriptor': 'Language Composite',
+      'subject_assess_type':'composite',
+      'subject_assessments': [
+        {'subject_assess_ref_1':[{'subject_asses_ref_colname':'txkea_vocabulary_1', 'subject_asses_ref_descriptor':'Vocabulary Wave 1'}]},
+        {'subject_assess_ref_2':[{'subject_asses_ref_colname':'txkea_listening_comp_1', 'subject_asses_ref_descriptor':'Listening Comprehension 1'}]}
+      ],
+      'subject_assess_lang': 'English',
+      'has_cp_col': False
+    },
```

Comment:
These subjects ("screener" and "composite") reference scores from other assessments that are part of CLI TX-KEA. Using the scores from these other assessments, it is able to evaluate a student's progress on literacy, language, and potential for dyslexia. In this earthmover.yml, I am treating these "screener" and "composite" subjects as their own unique categories of subjects with colnames "screener_assess_ref_x_score" and "composite_assess_ref_x_score". 

In the Assessments.jsont, these subjects are treated as subject assessments, with an added "type" descriptor designating them as "screener" and "composite" respectively. The CLI TX-KEA assessments they reference are treated as Objective Assessments. I am not sure if this is the best way to approach this, so I am open to suggestions:

```
["studentAssessmentIdentifier": "...kea_dyslexia_screener",
     ...
    "scoreResults": [
      {"assessmentReportingMethodDescriptor": "uri://www.cliengage.org/TX-KEA/AssessmentReportingMethodDescriptor#Score",
        "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Integer",
        "result": "2" },
      ...
      { "assessmentReportingMethodDescriptor": "uri://www.cliengage.org/TX-KEA/AssessmentReportingMethodDescriptor#Type",
        "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
        "result": "screener"}]
        
    "studentObjectiveAssessments": [
        { "objectiveAssessmentReference": {....},          
          "scoreResults": [
            {"assessmentReportingMethodDescriptor": "uri://www.cliengage.org/TX-KEA/AssessmentReportingMethodDescriptor#Score",
              "resultDatatypeTypeDescriptor":"uri://ed-fi.org/ResultDatatypeTypeDescriptor#Integer",
              "result": "2" },
            {"assessmentReportingMethodDescriptor": "uri://www.cliengage.org/TX-KEA/AssessmentReportingMethodDescriptor#Subject",
              "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
              "result": "Letter Sounds"}] },
        {"objectiveAssessmentReference": {...},          
          "scoreResults": [
            {"assessmentReportingMethodDescriptor": "uri://www.cliengage.org/TX-KEA/AssessmentReportingMethodDescriptor#Score",
              "resultDatatypeTypeDescriptor":"uri://ed-fi.org/ResultDatatypeTypeDescriptor#Integer",
              "result": "2"},
            {"assessmentReportingMethodDescriptor": "uri://www.cliengage.org/TX-KEA/AssessmentReportingMethodDescriptor#Subject",
              "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
              "result": "Blending"}]}],]
```